source/features/netiquette.tsx

```diff
@@ -26,40 +26,49 @@ function getCloseDate(): Date {
 
 const threeMonths = toMilliseconds({days: 90});
 
-function addConversationBanner(newCommentField: HTMLElement): void {
+export function shouldDisplayNotice(): boolean {
 	if (!isClosedOrMerged()) {
-		return;
+		return false;
 	}
 
 	const closingDate = getCloseDate();
-	if (timeAgo(closingDate) < threeMonths) {
-		return;
-	}
+	return timeAgo(closingDate) > threeMonths;
+}
 
+export function getNoticeText(): JSX.Element {
+	const closingDate = getCloseDate();
 	const ago = <strong>{twas(closingDate.getTime())}</strong>;
 	const newIssue = <a href={buildRepoURL('issues/new/choose')}>new issue</a>;
+	return (
+		<>
+			This {pageDetect.isPR() ? 'PR' : 'issue'} was closed {ago}. Please consider opening a {newIssue} instead of leaving a comment here.
```

Comment:
Not sure if showing this by default is a good idea. There are projects which actually do the opposite to reduce the amount of created issues and relevant people get a notification.

Reply:
We do what makes sense for most projects (personally I don't mind either way), and it is possible to bring back the comment field.

Also you are kinda commenting on the wrong place, as this pull request is for this repo (https://github.com/refined-github/refined-github) only.

Reply:
> Also you are kinda commenting on the wrong place, as this pull request is for this repo ([refined-github/refined-github](https://github.com/refined-github/refined-github)) only.

Than this part is borked, since this also shows up in other places like https://github.com/NixOS/nixpkgs/pull/205391

![image](https://user-images.githubusercontent.com/7258858/230774287-a034723a-62b4-4fa2-91c0-ba637b8cf194.png)

Reply:
@SuperSandro2000 to clarify:

### `netiquette` (everywhere, can be disabled)

It just adds a notice

<img width="677" alt="Screenshot 8" src="https://user-images.githubusercontent.com/1402241/227764715-ac788d65-f273-44a7-bed3-15766b1b434a.png">

### `rgh-netiquette` (just the `refined-github` organization, cannot be disabled)

It hides the field completely

<img width="677" alt="Screenshot 10" src="https://user-images.githubusercontent.com/1402241/227764733-dcc0d155-6218-4820-8f97-34edb06e1ae7.png">

Reply:
> There are 2 features, one that applies to all repositories and one that applies to only refined-github.
> 
> This PR is for the refined-github.

Yeah, I got this, that's not my point. I was thinking if this is not better disabled by default, but the more I think about it, the more I am divided and the more I think the unintrusive reminder is probably a good idea.